L/LBFGSB/build_tarballs.jl

```diff
@@ -0,0 +1,36 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "LBFGSB"
```

Comment:
It looks like the name of the upstream library is `L-BFGSB`, which we should spell `L_BFGSB` here to avoid dashes in the package name.

Reply:
```suggestion
name = "Lbfgsb"
```
It should be [L-BFGS-B](http://users.iems.northwestern.edu/~nocedal/lbfgsb.html). I think it's also good to use `Lbfgsb` as it's the libname in the original Makefile.

Reply:
> It should be L-BFGS-B.

Right, I missed the last B

> I think it's also good to use Lbfgsb as it's the libname in the original Makefile.

Usually we follow the name of the whole library as it's spelt upstream, not the name of a single product.